elevate-project/constants/routes.js

```diff
@@ -3049,5 +3049,28 @@ module.exports = {
             },
             service: "project"
         },
+        {
+            sourceRoute: "/project/health",
+            type: "GET",
+            inSequence: false,
+            orchestrated: false,
+            targetRoute: {
+                path: "/project/health",
```

Comment:
@MallanagoudaB keep target route to /health  for every service. make change in source route only